src/main/kotlin/eu/kevin/api/models/auth/authentication/response/StartAuthenticationResponse.kt

```diff
@@ -4,6 +4,6 @@ import kotlinx.serialization.Serializable
 
 @Serializable
 data class StartAuthenticationResponse(
-    val authorizationLink: String,
+    var authorizationLink: String,
```

Comment:
Why was this changed to `var`?

Reply:
I made both link mutable in order to allow the UI language changes: https://developer.kevin.eu/platform/payments/frame-customisation#change-the-ui-language

Reply:
`val localAuthorizationLink = response.authorizationLink + "?lang=en"` could work. I would want to keep properties immutable where possible since it reduces the cognitive load while programming

Reply:
What if I make optional `lang` param in kevin client so that authorization and confirm links would have `lang` query param attached by default if defined? We could then leave properties immutable

Reply:
Yes, but maybe it could be added to request models instead for better flexibility

Reply:
The `lang` quey param is not part of the request parameters. It must be appended on the received confirm or authorization link. So logically it does not fit on request model. But I am also confused how should I enable the UI language changes if response parameters are not mutable. Looking at the lib, I do not have much control on the plain json response.

Reply:
We don't have to pass `lang` to the actual HTTP request, just use it to update the authorization link in the response object. We can make a copy of the immutable response object, for example at the end of the method:
`.let { 
        it.copy(authorizationLink = it.authorizationLink + request.lang)
    }`